### DISCUSSION TITLE
rocksdb-doc-cn/doc/Partitioned-Index-Filters

### DISCUSSION THREAD

**Discussion by giscus** [wanghenshui/rocksdb-doc-cn#8] [General]
# rocksdb-doc-cn/doc/Partitioned-Index-Filters

中文版rocksdb文档，感谢johnzeng https://wanghenshui.github.io/rocksdb-doc-cn/

https://wanghenshui.github.io/rocksdb-doc-cn/doc/Partitioned-Index-Filters.html

**Comment by infdahai** [comment]
学习了，很好的资料。
